workflow_imaging/pipeline.py

```diff
@@ -31,9 +31,15 @@
 class Session(dj.Manual):
     definition = """
     -> subject.Subject
-    session_datetime: datetime
+    session_datetime: datetime(3)
     """
 
+    class Directory(dj.Part):
+        definition = """
+        -> master
+        session_dir: varchar(256)       # Relative path, relative to the `imaging_root_data_dir`
```

Comment:
are there multiple session directories? In `get_scan_image_files` it is used as though there can be only one, retrieved with `fetch1`.

Reply:
there is only one session directory per session, this is missing the `---`. Thanks for the catch Dimitri, fixing in next commits